knative-serving/external-ingress-routing/configuring-service-routes.adoc

```diff
@@ -12,4 +12,10 @@ If you want to configure a Knative service to use your TLS certificate on {ocp-p
 When you complete the following procedure, the default {ocp-product-title} route in the `knative-serving-ingress` namespace is not created. However, the Knative route for the application is still created in this namespace.
 ====
 
-include::modules/serverless-openshift-routes.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/serverless-openshift-routes.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+== Additional resources
+
+* See xref:../config-applications/configuring-revision-timeouts.adoc#configuring-default-route-timeouts-globally_configuring-revision-timeouts[Configuring the default route timeouts].
```

Comment:
Additional resources do not use an asterisk. See the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections). 
Perhaps a change for a later time. Not enough to block this merge.